java/fury-core/src/main/java/org/apache/fury/reflect/ReflectionUtils.java

```diff
@@ -380,6 +380,19 @@ public static List<Field> getFields(Class<?> cls, boolean searchParent) {
     return fields;
   }
 
+  public static List<Field> getFieldsWithoutSuperClasses(Class<?> cls, Set<Class<?>> superClasses) {
```

Comment:
You can use `org.apache.fury.reflect.ReflectionUtils#getFields(cls, false)` instead